packages/fx-core/templates/core/v3Migration/js.ts.app.local.yml

```diff
@@ -1,5 +1,78 @@
 version: 1.0.0
 
+{{#config.registerApp}}
+registerApp:
+  {{#if aad}}
+  - uses: aadApp/create
+    with:
+      name: $\{{CONFIG__MANIFEST__APPNAME__SHORT}}-aad
+      generateClientSecret: true
+
+  {{/if}}
+  {{#if teamsApp}}
+  - uses: teamsApp/create
+    with:
+      name: $\{{CONFIG__MANIFEST__APPNAME__SHORT}}
+  {{/if}}
+
+{{/config.registerApp}}
+{{#config.provision}}
+provision:
+  {{#if bot}}
+  - uses: botAadApp/create
+    with:
+      name: $\{{CONFIG__MANIFEST__APPNAME__SHORT}}-bot
+
+  - uses: botFramework/create
+    with:
+      botId: $\{{BOT_ID}}
+      name: $\{{CONFIG__MANIFEST__APPNAME__SHORT}}-bot
+      messagingEndpoint: $\{{PROVISIONOUTPUT__AZUREWEBAPPBOTOUTPUT__SITEENDPOINT}}/api/messages
```

Comment:
Use `namingConverterV3` to generate the placeholder.

Reply:
Good catch! As it depends on bicep content, let me use it in another PR.